### DISCUSSION TITLE
Ath0 setup

### DISCUSSION THREAD

**Discussion by ChristusArulanantham** [cypress-io/cypress#15358] [Questions and Help]
I followed the Auth0 setup documentation.

https://docs.cypress.io/guides/testing-strategies/auth0-authentication.html#Custom-Command-for-Auth0-Authentication

when I add the following to command.js 

the string does not seem to be an accepted value,

can you please help,

![image](https://user-images.githubusercontent.com/10866619/110250822-6528b580-7f75-11eb-8c61-eb0b0f46dfa0.png)

code is 

Cypress.Commands.add("loginByAuth0Api", (username: String, password: String) => {
  cy.log(`Logging in as ${username}`);
    const client_id = Cypress.env("auth0_client_id");
    const client_secret = Cypress.env("auth0_client_secret");
    const audience = Cypress.env("auth0_audience");
    const scope = Cypress.env("auth0_scope");
  
    cy.request({
      method: "POST",
      url: `https://${Cypress.env("auth0_domain")}/oauth/token`,
      body: {
        grant_type: "password",
        username,
        password,
        audience,
        scope,
        client_id,
        client_secret,
      },
    }).then(({ body }) => {
      const claims = jwt.decode(body.id_token);
      const { nickname, name, picture, updated_at, email, email_verified, sub, exp } = claims;
  
      const item = {
        body: {
          ...body,
          decodedToken: {
            claims,
            user: {
              nickname,
              name,
              picture,
              updated_at,
              email,
              email_verified,
              sub,
            },
            audience,
            client_id,
          },
        },
        expiresAt: exp,
      };
  
      window.localStorage.setItem('auth0Cypress', JSON.stringify(item));
  
      cy.visit("/");
    })
  })

**Comment by taras** [comment]
@ChristusArulanantham you copy and pasted TypeScript code into JavaScript. You need to remove `: String` text so it'll be proper JavaScript
